contrib/freeze.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/sh
-if [ $# -ne 1 ] ; then
-	echo Usage: $0 filename \(... will output to stdout\)
-	exit 1
+
+MAGIC=MAGIC_TTSIOD_RS
+RSBEP='rsbep -B 255 -D 223 -R 4080'
```

Comment:
I think this is generally not a good idea.

1. Variables are good almost exclusively for repeated use.
2. It relies on shell expansion which smells.
3. It obviates what `$RSBEP` command actually does.

Reply:
Compare freeze.sh and melt.sh side-to-side (I use `meld`) to see what I mean by 'unify the code'.

1. Not a 'var' but some 'config value' in this case. Didn't you notice it is right at the top?
2. Not sure what you mean. Too bad the script targets `/bin/sh`, not some `bash` so the command could be put in an array and used as `"${RSBEP[@]}"` instead of literal unquoted `$RSBEP`. Anyways, not an issue in **this** particular case
3. The idea is the rsbep cmd is the same in both scripts aside for `-d`. `$RSBEP` in one place and `$RSBEP -d` in another is quick to grok compared to `rsbep -B 255 -D 223 -R 4080` vs `rsbep -B 255 -D 223 -R 4080 -d`.

Reply:
1. Anything without `readonly` is a *var* :wink:. But seriously, these tiny scripts do not need any such "config". Ideally they would be merged into one script which would do both - freezing & melting.

2. Targeting `/bin/sh` is necessary cross-platformity. By shell expansion I meant creating a variable/invariable **deliberately** to be used **unquoted**. Which is a really bad practice. When I review such code I either reject it or in the worst case I require a huge comment warning about this intended usage (optionally with short reasoning why other safer approaches were not chosen).

3. dtto

-----
I should emphasize I am in no way an appointed reviewer here. I am mere observer of this repo for years and want to keep it as much functional and cross platform as possible.

Reply:
> Ideally they would be merged into one script which would do both - freezing & melting.

Good idea. I gonna cancel this pr and create another one with the tools merged. So that one would be kinda main and another just  a stub to call the other. No code repeats this way and no changes to building/packaging